src/tree-sitter/helpers.ts

```diff
@@ -17,7 +17,9 @@ export function makeParameterType(name: string, regexp: string | RegExp) {
 }
 
 export function toStringOrRegExp(s: string): string | RegExp {
-  const match = s.match(/^([/'"])(.*)([/'"])$/)
+  let match = s.match(/^([/'"])(.*)([/'"])$/)
+  // match multi-line comments
```

Comment:
As we add more languages, I think this kind of branching can get a bit messy. No need to fix in this PR, just adding a note as a reminder to refactor this.